site/_collections/feeds.js

```diff
@@ -14,30 +14,75 @@
  * limitations under the License.
  */
 
+const {defaultLocale} = require('../_data/site.json');
+const {i18n} = require('../_filters/i18n');
+
 /**
  * Returns an array of `FeedsCollectionItem` to generate the RSS feeds.
  *
  * @param {EleventyCollectionObject} collection
  * @returns {FeedsCollection} Key value pair of tag name with `FeedsCollectionItem`.
  */
 module.exports = collection => {
+  const MAX_POSTS = 10;
+  const posts = collection
+    .getAllSorted()
+    .reverse()
+    .filter(i => i.data.locale === defaultLocale);
+  const tags = ['devtools'];
   /** @type FeedsCollection */
   const feeds = {};
-  const tags = ['devtools'];
+  const blog = [];
+
+  for (const post of posts) {
```

Comment:
Would it also work to do something like:

```
for (const tag of tags) {
  const posts = collection.getFilteredByTag(tag).slice(MAX_POSTS);
  feeds[tag] = {
    items: posts,
    permalink: ...
    ...
  };
}
```

I realize it's technically not as efficient, but it does seem like it might be easier to read?

Reply:
@robdodson Something like this would technically work, though it'll have us looping through all the posts for each tag. Would you like me to possibly break it up and use some well named functions to try to make the code more readable?

Reply:
nah it's not a big deal